utils/tiles.py

```diff
@@ -0,0 +1,77 @@
+from ModestMaps.Core import Coordinate
+from mercantile import tiles
+from mercantile import tile
+#from typing import Iterator
```

Comment:
remove commented imports

Reply:
> remove commented imports

sure. But some background, but why it is here is that in my local python3 environment, I used it for type checking, it is way better experience in terms of development.

However in our runtime environment, python2, it is not supported so I commented them out, I can completely remove them, that's fine.

Reply:
you'll have to show me what you mean!  This sounds interesting

Reply:
I already removed the lines in code. 

But what I meant is this, see the following screenshot. The function signature and return type is highlighted on the IDE in green background because it recognized the typing information. However List and Iterator are yellow because it cannot recognize it because I removed the `from typing import Iterator`. The typing library is only available in python3, so even if I leverage typing highlighting during development, I have to comment them out since our runtime is on python2 and will throw an error.

In short, these import statements will be used for typing in python3.

![Snip20210519_4](https://user-images.githubusercontent.com/62725319/118892525-deb61b80-b8b5-11eb-99c7-672280ef3d66.png)